contracts/src/core/ERC20Mock.sol

```diff
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: MIT
 // OpenZeppelin Contracts (last updated v4.8.0) (token/ERC20/ERC20.sol)
 
-pragma solidity ^0.8.9;
+pragma solidity =0.8.12;
```

Comment:
is this change ok? seems unrelated to the PR

Reply:
It is, but make bindings is not working on main, I solved some small issues

Reply:
great, thanks